array-manipulation-1/src/head.js

```diff
@@ -0,0 +1,13 @@
+/* exported head */
+
+// look at each element of the array.
+// if it's the first element of the array,
+// then return that element
+
+function head(array) {
+  for (var i = 0; i < array.length; i++) {
+    if (i === 0) {
+      return array[i];
+    }
+  }
```

Comment:
This loop is not necessary. You already know what index the value is at, you should just retrieve it and `return` it.